server/fleet/hosts.go

```diff
@@ -407,6 +408,7 @@ var mdmNameFromServerURLChecks = map[string]string{
 	"airwatch":  WellKnownMDMVMWare,
 	"microsoft": WellKnownMDMIntune,
 	"simplemdm": WellKnownMDMSimpleMDM,
+	"fleetdm":   WellKnownMDMFleet,
```

Comment:
We should probably revisit how we match the MDM server URL for Fleet MDM. The approach here is consistent with the way things are currently set up in other parts of the code. But should this instead be matching against the user-configured server setting from the app config?

Reply:
this is accounted for in Luca's refactor here: https://github.com/fleetdm/fleet/pull/9133/files#diff-9aebb07c5e9eae43b4cf167ef1ac12d720a613da219e5342d9a8e992e5fd0815R1129-R1134

so in theory we don't need the `fleetdm` key here.